migrate_passwords.py

```diff
@@ -0,0 +1,62 @@
+''' Python script to migrate database password hashing algorithms. '''
```

Comment:
Can you add instructions for running this (in case we need to run a migration in the future)?